python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -152,6 +152,8 @@ def get$Name(self):
         ("varianceCol", "column name for the biased sample variance of prediction.",
          None, "TypeConverters.toString"),
         ("aggregationDepth", "suggested depth for treeAggregate (>= 2).", "2",
+         "TypeConverters.toInt"),
+        ("parallelism", "number of threads to use when fitting models in parallel.", "1",
```

Comment:
This should probably indicate that the value range is "(>= 1)"  at the end of the description.